### PR TITLE
Bugfix communityname document title & tooltip

### PR DESCRIPTION
**Pull request by Revadike**
Resolves https://github.com/tfedor/AugmentedSteam/issues/415 and resolves https://github.com/tfedor/AugmentedSteam/issues/389

### CODE REVIEW

File: css/enhancedsteam.css [hunk 1]
```diff
@@ -1685,6 +1685,13 @@ input[type=checkbox].es_dlc_selection:checked + label {
 	font-size: 12px;
 }
 
+/***************************************
+ * Regional Price Bugfix
+ **************************************/
+ /* .ds_options_tooltip {
+	 z-index: 99999 !important;
+ } */
+
```
Comment (MxtOUT) [inline css/enhancedsteam.css]: What should this be?

Comment (Revadike) [reply css/enhancedsteam.css]: https://github.com/tfedor/AugmentedSteam/issues/389

Comment (Revadike) [reply css/enhancedsteam.css]: Wait, why is this commented out?

**Comment by MxtOUT** [comment]
Waiting for an answer in #389

**Pull request merged**